src/main.rs

```diff
@@ -0,0 +1 @@
+fn main() {}
```

Comment:
When do we need this?
Could you please elaborate on this?

Reply:
hpx-rs wouldn't compile if we don't have a main fn inside `hpx-rs/src/main.rs` and if we omit `hpx-rs/src/main.rs` file rust-analyzer would show issues with project.
For the time being this is just to avoid errors while building.